actors/evm/src/interpreter/address.rs

```diff
@@ -158,4 +162,27 @@ mod tests {
             vec![0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01].as_slice() // ID address (u64 big endian) (8 bytes)
         ] => None,
     }
+
+    #[test]
+    #[allow(unused)]
```

Comment:
why allow unused?

Reply:
Err value of the TryFrom trait for EthAddress to Address: "output::StatusCode must be used" 
https://github.com/filecoin-project/builtin-actors/blob/36ec85aa6e3735447a7fb0dfab2ffcba83906d20/actors/evm/src/interpreter/output.rs#L31

Reply:
Hm. It's better to write that as `let _ = ...`. Otherwise, you can miss other error cases.